### PR TITLE
[ZT] WARP changelog

### PR DESCRIPTION
**Pull request by ranbel**
PCX-15144

### CODE REVIEW

File: src/content/changelogs/warp.yaml [hunk 1]
```diff
@@ -5,6 +5,186 @@ productLink: "/cloudflare-one/connections/connect-devices/warp/"
...
+      Known issues:
+        - DNS resolution may be broken when all of the following conditions are true:
+          - WARP is in Secure Web Gateway without DNS filtering (tunnel-only) mode.
+          - A custom DNS server address is configured on the primary network adapter.
+          - The custom DNS server address on the primary network adapter is changed while WARP is connected.
+
+          To work around the DNS issue, reconnect the WARP client by toggling off and back on.
+
+  - publish_date: "2024-12-13"
+    title: WARP client for Windows (version 2024.11.688.1)
```
Comment (kokolocomotion1) [inline src/content/changelogs/warp.yaml]: My bad, the version for the windows beta published 2024-12-13 is version 2024.12.326.1

File: src/content/changelogs/warp.yaml [hunk 2]
```diff
@@ -5,6 +5,186 @@ productLink: "/cloudflare-one/connections/connect-devices/warp/"
...
+    description: |-
+      A new GA release for the iOS Cloudflare One Agent is now available in the [iOS App Store](https://apps.apple.com/us/app/cloudflare-one-agent/id6443476492). This release includes support for an exciting new capability, [per-app VPN](/cloudflare-one/connections/connect-devices/warp/deployment/mdm-deployment/partners/intune/#per-app-vpn-for-ios). This release also includes fixes and minor improvements.
+
+      Changes and improvements:
+        - Added support for per-app VPN configuration.
+        - Fixed issue where some users could not connect unless they rotated their keys after an update.
+        - Fixed a potential crash when connecting to the tunnel.
+
+  - publish_date: "2024-12-19"
+    title: Cloudflare One Agent for Android (version 2.3 (4616))
```
Comment (kokolocomotion1) [inline src/content/changelogs/warp.yaml]: actually let's leave teh version description as just:  version 2.3

**Review by kokolocomotion1 — CHANGES_REQUESTED** [review]
Hey I left just two small changes needed. otherwise good to go. thanks Ranbel!

File: src/content/changelogs/warp.yaml [hunk 3]
```diff
@@ -5,6 +5,186 @@ productLink: "/cloudflare-one/connections/connect-devices/warp/"
...
+        - DNS resolution may be broken when all of the following conditions are true:
+          - WARP is in Secure Web Gateway without DNS filtering (tunnel-only) mode.
+          - A custom DNS server address is configured on the primary network adapter.
+          - The custom DNS server address on the primary network adapter is changed while WARP is connected.
+
+          To work around the DNS issue, reconnect the WARP client by toggling off and back on.
+  - publish_date: "2024-12-19"
+    title: WARP client for macOS (version 2024.12.554.0)
+    description: |-
+      A new GA release for the macOS WARP client is now available in the [App Center](https://install.appcenter.ms/orgs/cloudflare/apps/1.1.1.1-macos-1/distribution_groups/release). This release contains improvements to support custom Gateway certificate installation in addition to the changes and improvements included in version 2024.12.492.0(20241218.36).
```
Comment (ranbel) [inline src/content/changelogs/warp.yaml]: @kokolocomotion1 should we keep the version # that is in parentheses? `(20241218.36)`

File: src/content/changelogs/warp.yaml (hunk 2 above)
Comment (ranbel) [inline src/content/changelogs/warp.yaml]:
```suggestion
    title: Cloudflare One Agent for Android (version 2.3)
```

File: src/content/changelogs/warp.yaml (hunk 1 above)
Comment (ranbel) [inline src/content/changelogs/warp.yaml]:
```suggestion
    title: WARP client for Windows (version 2024.12.326.1)
```

File: src/content/changelogs/warp.yaml (hunk 3 above)
Comment (kokolocomotion1) [reply src/content/changelogs/warp.yaml]: @ranbel let's not include it so we keep the same formatting as our past changelogs

**Review by kokolocomotion1 — APPROVED** [review]

File: src/content/changelogs/warp.yaml [hunk 4]
```diff
@@ -5,6 +5,186 @@ productLink: "/cloudflare-one/connections/connect-devices/warp/"
...
```
Comment (ranbel) [inline src/content/changelogs/warp.yaml]:
```suggestion
      A new GA release for the macOS WARP client is now available in the [App Center](https://install.appcenter.ms/orgs/cloudflare/apps/1.1.1.1-macos-1/distribution_groups/release). This release contains improvements to support custom Gateway certificate installation in addition to the changes and improvements included in version 2024.12.492.0.
```

**Pull request merged**